docs/vega-core.js

```diff
@@ -927,7 +927,7 @@
   }
   /**
    * URI sanitizer function.
-   * @param {string} uri - The uri (url or filename) to sanity check.
+   * @param {string} uri - The uri (url or filename) to check.
```

Comment:
We usually only add compiled files in release prs. Please remove them from this pull request.